### DISCUSSION TITLE
Waveforms not appearing

### DISCUSSION THREAD

**Discussion by Axilotl17** [ppy/osu#18200] [Q&A] [answered]
In Lazer 2022.509.0 the waveforms of songs in the editor appear to not appear.

map for example
[Nanahira - Asu wo Tsunaide (Axilotl).zip](https://github.com/ppy/osu/files/8656380/Nanahira.-.Asu.wo.Tsunaide.Axilotl.zip)

**Comment by Axilotl17** [comment]
running arch linux if that tells you anything

**Comment by smoogipoo** [comment, marked as answer]
Please provide logs: https://github.com/ppy/osu/discussions/15188

**Reply by Axilotl17** [reply]
nevermind i fixed it by changing skins /shrug

**Reply by frenzibyte** [reply]
Can you please provide what skin was causing this issue? That doesn't sound normal at all.

**Reply by Axilotl17** [reply]
sorry - i just changed skins back and fourth and it fixed. both skins worked fine.

**Reply by Theighlin** [reply]
#18479 for reference
